### ISSUE TITLE
<NAME THE FEATURE> - <SUMMARIZE THE PROBLEM>

### ISSUE THREAD

**Issue by sidewayss** [mdn/browser-compat-data#21307]
### What type of issue is this?

Incorrect support data (example: BrowserX says "86" but support was added in "40")

### What information was incorrect, unhelpful, or incomplete?

Based on my tests on the latest versions of the OSes and browsers, navigator.userAgentData is not available on any browser in MacOS or iOS, including Chrome.  Here is the relevant section of the MDN page:

https://developer.mozilla.org/en-US/docs/Web/API/Navigator/userAgentData#browser_compatibility

Here is the caniuse.com page: https://caniuse.com/?search=navigator.useragentdata

I don't know how to make those edits here, or what edits to make. Should it be a numbered footnote? Should the green color on caniuse.com be that olive green instead of the bright green? How is that indicated in the JSON here? Chrome is the only browser that is incorrectly labeled, but maybe it should be a more global note.  fyi - I have contributed to MDN/content, but not to this project, so I'm interested in how things work here as well.

### What browsers does this problem apply to, if applicable?

Chromium (Chrome, Edge 79+, Opera, Samsung Internet)

### What did you expect to see?

I expect to see a footnote about Chrome on iOS/MacOS or a global note of some kind about those operating systems.

### Did you test this? If so, how?

I tested it on browserstack.com using the latest iOS, MacOS and Chrome.

### Do you have anything more you want to share?

I can share screenshots of my Watch results in the debugger if you like.

**Comment by sidewayss** [comment]
Posting a follow-up comment to see if I can re-open this issue...

**Comment by sidewayss** [comment]
Not sure why it was rejected in the first place, but I added content to MDN urls sections that I assumed would be filled in automatically (as stated in the new issue form).

**Comment by queengooborg** [comment]
Hey @sidewayss -- your issue was automatically closed because the title was left as the default.  We get a lot of spam issues opened, and almost always, they leave the title untouched, so this bot is intended to be a stopgap against it.

Of course, not all issues with default titles are spam, so the bot indicates what the next steps are...or tries, at least.  I'll tweak the bot's message to better relay what's needed!

**Comment by sidewayss** [comment]
@queengooborg - Yea, I realized that fact only after I had left this for closed and had created a new issue #21308.  I think I was distracted by the fact that the title was pre-filled more than the bot's message, which in hindsight I could have read more carefully.
